kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UserTaskServicesClient.java

```diff
@@ -35,6 +35,8 @@
 
     void completeTask(String containerId, Long taskId, String userId, Map<String, Object> params);
 
+    void claimStartCompleteTask(String containerId, Long taskId, String userId, Map<String, Object> params);
+
```

Comment:
well, I thought that it would be clearer to call it claimStartCompleteTask - since it exactly says what it will do .... completeTaskAutoProgress is quite abstract but if it is ok and you think it's better, I will change it.